dll/keyboard/kbdcz/kbdcz.c

```diff
@@ -98,9 +98,6 @@ ROSDATA USHORT scancode_to_vk[] = {
   /* - 77 - */
   VK_EMPTY, VK_EMPTY, VK_EMPTY, VK_EMPTY,
   VK_EMPTY, VK_EMPTY, VK_EMPTY, VK_EMPTY, /* PA1 */
-  VK_EMPTY,
```

Comment:
Ditto

Reply:
Same as above, I just removed excessive values (there should be 127 maximum), and replaced zeros with VK_EMPTY.